### PR TITLE
View first version mobile-first

### PR DESCRIPTION
**Pull request by YannRaulet**

### CODE REVIEW

**Review by MorganeGrenes — DISMISSED** [review]

**Review by rickifox — DISMISSED** [review]
It's OK to me

**Review by axelhamil — DISMISSED** [review]
All it's ok.

File: config/config.php [hunk 1]
```diff
@@ -17,4 +17,4 @@
 define('APP_VIEW_PATH', __DIR__ . '/../src/View/');
 define('APP_CACHE_PATH', __DIR__ . '/../temp/cache/');
 
-define('HOME_PAGE', 'home/index');
+define('HOME_PAGE', 'Livre/index');
```
Comment (RomainClair) [inline config/config.php]: It should be livre (with a lowercase "l")

**Review by RomainClair — CHANGES_REQUESTED** [review]
Don't add everything in each PR.

PR must be atomic : one US/Feature

**Review by MorganeGrenes — APPROVED** [review]

**Review by MorganeGrenes — APPROVED** [review]

**Review by rickifox — APPROVED** [review]

**Pull request closed without merging**